File: src/breslin/clientside/command/command.cpp
#include "command.h"

Command::Command()
{
	mPosition = new Vector3D();
	mRotation = new Vector3D();
	mVelocity = new Vector3D();
	mFrameTime = 0;

	mRotationSpeed = 0.0f;	
	mScore = 0;
	mDeltaCode = 0;
}

Command::~Command()
{
}

File: src/breslin/clientside/command/Command.java
package breslin.clientside.command;


/**********************************
*          INCLUDES
**********************************/
//math
import breslin.math.Vector3D;

public class Command
{

public Command()
{
	mPosition     = new Vector3D();
	mRotation = new Vector3D();
	mVelocity          = new Vector3D();
	mRotationSpeed = 0.0f;
	mFrameTime = 0;
}

/**************************************************
*			VARIABLES
**************************************************/
public Vector3D mPosition;
public Vector3D mRotation;
public Vector3D mVelocity;	
public float mRotationSpeed;
public int mFrameTime;
/**************************************************
*			METHODS
**************************************************/

}
